src/Analysis/Codelyzer.Analysis.Model/IfStatement.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace Codelyzer.Analysis.Model
+{
+    public class IfStatement : UstNode
+    {
+        [JsonProperty("base-type", Order = 10)]
+        public string BaseType { get; set; }
+
+        [JsonProperty("base-type-original-def", Order = 11)]
+        public string BaseTypeOriginalDefinition { get; set; }
+
+        [JsonProperty("base-list", Order = 12)]
+        public List<string> BaseList { get; set; }
+
+        [JsonProperty("modifiers", Order = 20)]
+        public string Modifiers { get; set; }
+
+        [JsonProperty("references", Order = 99)]
+        public Reference Reference { get; set; }
+        public string SemanticAssembly { get; set; }
```

Comment:
Does the IfStatement syntax node have BaseType, BaseTypeOriginalDefinition, BaseList, Modifiers, or Reference defined? If not, we should remove these properties since they'll always be null.

We might need to revisit the properties of many of the new UstNode classes (and their respective Handlers) as well, since many of them have the same properties defined.

Reply:
+1